validator/client/iface/validator.go

```diff
@@ -68,6 +68,8 @@ type Validator interface {
 	SetGraffiti(ctx context.Context, pubKey [fieldparams.BLSPubkeyLength]byte, graffiti []byte) error
 	DeleteGraffiti(ctx context.Context, pubKey [fieldparams.BLSPubkeyLength]byte) error
 	HealthTracker() *beacon.NodeHealthTracker
+	RetrieveHost() string
+	UpdateHost(host string)
```

Comment:
Can these be called `Host`/`ChangeHost` for consistency

Reply:
+1